main.go

```diff
@@ -101,18 +126,75 @@ func initLogging() (*os.File, error) {
 	// Sample out: 2020/07/31 21:47:19.153535 main.go:145: Version: 1.0.0.0 ERROR: Hello World!
 	errorLogger = log.New(io.MultiWriter(file, os.Stderr), "Version: "+version+" ERROR: ", loggerFlags)
 
+	defer file.Close()
```

Comment:
This should be before the error check so that we ensure that we're always closing the file on leaving the function